platform/flash/zephyr/src/mender-flash.c

```diff
@@ -126,20 +126,17 @@ mender_flash_abort_deployment(void *handle) {
 mender_err_t
 mender_flash_confirm_image(void) {
 
-    int          result;
-    mender_err_t ret = MENDER_OK;
-
     /* Validate the image if it is still pending */
     if (false == mender_flash_is_image_confirmed()) {
```

Comment:
Please also remove this boolean comparison, it looks lame.

Reply:
It is pretty lame indeed